packages/charts/react-charting/src/components/LineChart/LineChart.base.tsx

```diff
@@ -278,6 +278,7 @@ export class LineChartBase extends React.Component<ILineChartProps, ILineChartSt
         {...this.props}
         chartTitle={this._getChartTitle()}
         points={points}
+        supportNegativeData={this.props.supportNegativeData}
```

Comment:
> supportNegativeData={this.props.supportNegativeData} [](http://example.com/codeflow?start=7&length=53)

this is not needed. Already being passed with the spread operator in the beginning